### DISCUSSION TITLE
Is it `*.WinUI.*` or `*.WinUI.UI.*` ?

### DISCUSSION THREAD

**Discussion by Jay-o-Way** [CommunityToolkit/Windows#308] [Q&A] [answered]
Take a look at the following image. On the bottom three, we see both `WinUI.Converters` and `WinUI.UI.Converters`. I also saw this for `.Controls` [here](https://github.com/CommunityToolkit/Windows/issues/69#issuecomment-1566212714). What is correct?

![image](https://github.com/microsoft/PowerToys/assets/65828559/96f3e861-e177-4dd0-b583-3cf3428a378f)

**Comment by niels9001** [comment, marked as answer]
CommunityToolkit.WinUI.* is the new namespace used for 8.x. We are dropping the .UI. as we simplified the namespaces!
